### PR TITLE
FINAL:  ApiEndpoint Refactor: cycleData/descriptions, messageCenter and section -> sectionName

### PR DESCRIPTION
**Pull request by sorja**
part of #1458

### CODE REVIEW

**Review by rfontanarosa — APPROVED** [review]

File: client/store/pages/assessmentSection/actions/getDescription.ts [hunk 1]
```diff
@@ -2,21 +2,21 @@ import { ApiEndPoint } from '@common/api/endpoint'
...
-  } = await axios.get(ApiEndPoint.Assessment.Data.descriptions(), {
-    params: { countryIso, assessmentName, cycleName, sectionName, name },
-  })
+export const getDescription = createAsyncThunk<Record<string, string>, CycleDataParams & { name: string }>(
+  'section/get/description',
+  async ({ countryIso, assessmentName, cycleName, sectionName, name }) => {
+    const {
+      data: { content },
+    } = await axios.get(ApiEndPoint.CycleData.descriptions(), {
+      params: { countryIso, assessmentName, cycleName, sectionName, name },
```
Comment (minotogna) [inline client/store/pages/assessmentSection/actions/getDescription.ts:13]: params is still section.

File: meta/api/request/index.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export type CycleParams = {
 export type CycleDataParams = CycleParams & {
   // TODO: rename to sectionName
   section: string
+  sectionName: string
```
Comment (minotogna) [inline meta/api/request/index.ts:15]: for now, param must be section.
in all calls internally (api / controllers we use sectionName)
renaming this parameter is a separate task,

File: server/api/assessment/index.ts [hunk 3]
```diff
@@ -45,8 +45,8 @@ export const AssessmentApi = {
     // requireView: We don't pass table for sections - always allow read
     express.get(ApiEndPoint.Assessment.sections(), AuthMiddleware.requireView, getSections)
     express.get(ApiEndPoint.Sections.metadata(), AuthMiddleware.requireView, getSectionMetadata)
-    express.get(ApiEndPoint.Assessment.Data.descriptions(), AuthMiddleware.requireView, getDescription)
-    express.put(ApiEndPoint.Assessment.Data.descriptions(), AuthMiddleware.requireEdit, upsertDescription)
+    express.get(ApiEndPoint.CycleData.descriptions(), AuthMiddleware.requireView, getDescription)
+    express.put(ApiEndPoint.CycleData.descriptions(), AuthMiddleware.requireEdit, upsertDescription)
```
Comment (minotogna) [inline server/api/assessment/index.ts:49]: to move under api/cycleData (and files )

**Review by minotogna — CHANGES_REQUESTED** [review]

File: meta/api/request/index.ts (hunk 2 above)
Comment (sorja) [reply meta/api/request/index.ts:15]: most of the work already done in this and next
urlParam section needs to be renamed to sectionName

Comment (minotogna) [reply meta/api/request/index.ts:15]: the middleware uses section. use as url param section for now.
we'll rename it in a next task

**Review by minotogna — APPROVED** [review]

**Pull request merged**